### PR TITLE
Add missing methods to docs

### PR DESCRIPTION
**Pull request by samgermain**
Solves https://github.com/ccxt/ccxt/issues/12140

- [x] addMargin
- [x] cancelOrders
- [x] createLimitOrder
- [x] deposit
- [x] editOrder
- [x] fetchAccounts
- [x] fetchBorrowRate
- [x] fetchBorrowRateHistory
- [x] fetchBorrowRatesPerSymbol
- [x] fetchBorrowRates
- [x] fetchCanceledOrders
- [x] fetchClosedOrder
- [x] fetchDeposit
- [x] fetchDepositAddressesByNetwork
- [x] fetchDeposits
- [x] fetchFundingFee
- [x] fetchFundingHistory
- [x] fetchLedgerEntry
- [x] fetchOpenOrder
- [x] fetchOrderBooks
- [x] fetchPosition
- [x] fetchPositionsRisk
- [x] fetchTradingFee
- ❌  fetchTradingLimits
- [x] fetchTransfers
- [x] fetchWithdrawal
- [x] reduceMargin
- [x] setLeverage
- [x] setMarginMode
- ❌ setPositionMode
- [x] signIn
- [x] withdraw

### CODE REVIEW

**Comment by samgermain** [comment]
@kroitor This is all done now

**Comment by samgermain** [comment]
@kroitor Is there anything else needed here? I want to start writing more docstrings, and linking to stuff in this documentation

**Comment by kroitor** [comment]
@samgermain we want to make sure that we don't break the existing formatting for RTD. Namely, we should put a blank line before and after each header and also before and after each list.

**Comment by samgermain** [comment]
> @samgermain we want to make sure that we don't break the existing formatting for RTD. Namely, we should put a blank line before and after each header and also before and after each list (plz keep the formatting uniform with the doc around)

Ok, done

File: wiki/Manual.md [hunk 1]
```diff
@@ -2463,50 +2554,50 @@ The possible values in the `status` field are:
...
+- `fetchMarketLeverageTiers()` (single symbol)
+- `fetchLeverageTiers([symbol1, symbol2, ...])` (multiple symbols) 
+- `fetchLeverageTiers()` (all market symbols) 
 
 ```Javascript
 fetchMarketLeverageTiers(symbol, params = {})
 ```
 
-The `fetchLeverageTiers()` method can be used to obtain the maximum leverage for a market at varying position sizes. It can also be used to obtain the maintenance margin rate, and the max tradeable amount for a market when that information is not available from the market object:
+Parameters
```
Comment (kroitor) [inline wiki/Manual.md:2577]: I would call these "Arguments" to avoid confusion with `params`.

File: wiki/Manual.md [hunk 2]
```diff
@@ -2601,68 +2702,167 @@ The `fetchFundingRate()` method will return a structure like shown below:
...
+- **symbol** (String) Unified CCXT symbol (e.g. `"BTC/USDT:USDT"`)
+- **since** (Integer) Timestamp for the earliest funding rate (e.g. `1645807945000`)
+- **limit** (Integer) The maximum number of funding rates to retrieve (e.g. `10`)
+- **params** (Dictionary) Extra parameters specific to the exchange API endpoint (e.g. `{"endTime": 1645807945000}`)
 
-The `fetchFundingRates()` method will return a structure like shown below:
+Returns
 
+- An array of [funding rate history structures](#funding-rate-history-structure)
+### Funding Rate History Structure
```
Comment (kroitor) [inline wiki/Manual.md:2722]: Need whitespacing around headers

File: wiki/Manual.md [hunk 3]
```diff
@@ -2601,68 +2702,167 @@ The `fetchFundingRate()` method will return a structure like shown below:
...
+    marginRatio: 0.4881,
+    datetime: "2022-03-16T08:45:54.000Z",
+    marginType: 'cross',
+    side: "long",
+    hedged: false,
+    percentage: 78
+}
 ```
 
+
```
Comment (kroitor) [inline wiki/Manual.md:2861]: This whitespace line is excessive.

File: wiki/Manual.md [hunk 4]
```diff
@@ -2826,6 +3031,23 @@ $exchange = new $exchange_class (array (
...
+Parameters
+
+- **params** (Dictionary) Parameters specific to the exchange API endpoint (e.g. `{"2fa": "329293"}`)
+
+Returns
+
+- response from the exchange
+
+
```
Comment (kroitor) [inline wiki/Manual.md:3050]: We need one whitespace line, before and after each header.

File: wiki/Manual.md [hunk 5]
```diff
@@ -2826,6 +3031,23 @@ $exchange = new $exchange_class (array (
...
 
+### Sign In
+
+Some exchanges required you to sign in prior to calling private methods, which can be done using the `signIn` method
+
+```Javascript
+signIn (params = {})
+```
+
+Parameters
```
Comment (kroitor) [inline wiki/Manual.md:3042]: I would make it an `#### Arguments` subheader throughout the docs.

Comment (samgermain) [reply wiki/Manual.md:3042]: Function parameters are the names listed in the function's definition. Function arguments are the real values passed to the function. Parameters are initialized to the values of the arguments supplied.

Comment (kroitor) [reply wiki/Manual.md:3042]: Yes, you're right.

File: wiki/Manual.md [hunk 6]
```diff
@@ -1534,6 +1545,42 @@ print(dir(ccxt.hitbtc()))           # Python
 var_dump (new \ccxt\okcoinusd ()); // PHP
 ```
 
+## Accounts
```
Comment (kroitor) [inline wiki/Manual.md:1548]: Why would we put the description of `fetchAccounts` into the intro section though?

**Pull request merged**